colossalai/fx/profiler/constants.py

```diff
@@ -30,3 +30,10 @@
 CLONE_ATEN = [
     aten.clone.default,
 ]
+
+RELU_LIKE_OPS = [
+    torch.nn.functional.relu,
+    torch.nn.functional.softmax,
+]
```

Comment:
Still you should define a term for this kind of ops, `relu_like` does not suggest any useful info.

Reply:
Or call them something like `OUTPUT_SAVED_OPS`?

Reply:
I put a docstring in the function which used this constant to illustrate.

Reply:
Yup I see it. Just think it could be clearer. I can accept it at the moment but hope it can be improved.

Reply:
Okay, I will come back to refactoring the profiler after @Cypher30 finished the backward estimation. I already have some ideas to remove some of these constants without affecting the performance.